### PR TITLE
Remove obsolete sample code and Rename Views Folder to View Location

### PR DESCRIPTION
**Pull request by srkeshav**

### CODE REVIEW

File: input/docs/handbook/commands/binding-commands.md [hunk 1]
```diff
@@ -67,3 +59,5 @@ this.BindCommand(
     x => x.myControl,
     x => x.SomeProperty);
 ```
+**Platform Specific **
+WPF: Arguments passed through CommandParameter in the view are automatically bound to `TInput` in `ReactiveCommand<Input, Unit>`.
```
Comment (glennawatson) [inline input/docs/handbook/commands/binding-commands.md]: This one isn't WPF specific, also happens on UWP is another platform I can think of.

Comment (glennawatson) [reply input/docs/handbook/commands/binding-commands.md]: Also might be worthwhile doing more of an example of this one, and also how the preferred approach due to platform compatibility is probably not to use that functionality but instead prefer View/VM binding of the parameters.

Comment (glennawatson) [reply input/docs/handbook/commands/binding-commands.md]: Also appears as if Xamarin Forms also support the CommandParameter approach https://forums.xamarin.com/discussion/85338/how-to-pass-value-from-binding-in-the-commandparameter-from-a-button

Comment (srkeshav) [reply input/docs/handbook/commands/binding-commands.md]: Yes, CommandParameter is supported in Xamarin Forms as well, IIRC.

For views generated through viewmodels, we could do VM Binding but for static design, the ViewModel binding might be an unneeded complexity (Introducing a VM property).

Comment (glennawatson) [reply input/docs/handbook/commands/binding-commands.md]: Potentially, we just do recommendations and leave it up to the user to decide based on their use case, so you can mention those two scenarios.

File: input/docs/handbook/commands/binding-commands.md [hunk 2]
```diff
@@ -41,17 +41,9 @@ Here, the `SomeEvent` on `myControl` will be used to trigger command execution i
 
 > **Note** When using this overload inside `WhenActivated`, it's important to dispose the binding when deactivating the view. `BindCommand` will subscribe to the given event each time the view is activated, if the binding is not disposed it will not unsubscribe from the event. This will lead to multiple subscriptions to the event, which will make the command execute once for each of the event subscriptions.
 
-Finally, `BindCommand` also provides overloads that allow you to specify a parameter with which to execute the command. The parameter can be provided as a function, an observable, or even an expression that resolves a property on the view model:
+`BindCommand` also provides overloads that allow you to specify a parameter with which to execute the command. The parameter can be provided as a function, an observable, or even an expression that resolves a property on the view model:
```
Comment (glennawatson) [inline input/docs/handbook/commands/binding-commands.md]: I would remove the word 'even' and change it to 'or to an expression'

File: input/docs/handbook/commands/binding-commands.md [hunk 3]
```diff
@@ -67,3 +59,102 @@ this.BindCommand(
...
+         //Here FeedType will be the value assigned when the ViewModel was created
+    }
+ }
+ ```
+
+</details>
+
+
+`CommandParameter` can be a good choice when the view contains multiple buttons that cannot be generated using an `ItemTemplate`. 
+Creating ViewModel properties for such buttons adds an unnecessary overhead in the design of the ViewModel and is best avoided.
```
Comment (glennawatson) [inline input/docs/handbook/commands/binding-commands.md]:
```suggestion
`CommandParameter` is a way to reuse a command without having to create additional properties on the ViewModel. Be aware that `CommandParameter` cannot be used to determine if a command can execute or not.
```

File: input/docs/handbook/commands/binding-commands.md (hunk 2 above)
Comment (glennawatson) [reply input/docs/handbook/commands/binding-commands.md]:
```suggestion
`BindCommand` also provides overloads that allow you to specify a parameter with which to execute the command. The parameter can be provided as a function, an observable, or an expression that resolves a property on the view model:
```

**Pull request merged**